lesson01.trivia/index.html

```diff
@@ -1,22 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
+<html>
 <head>
-    <title>Simple Trivia</title>
     <meta charset="utf-8">
-    <meta name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <style>
-        body {
+    <link href="style.css" type="text/css" rel="stylesheet">
+    <title>Trivia</title>
+     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-        }
-    </style>
 </head>
 <body>
-    <header>
-        <h1>Simple Trivia</h1>
-    </header>
-    <main>
-        Content Here
-    </main>
+ 
+     <div class="firstBlock" align="center" >
+               <h1 id="title">TRIVIA</h1>  
+     </div>   
+     <div class="block" align="center">
+       <!-- <div align="center"> -->
+            <p id="question" align="center" >What is the color of panda? </p>
+            <button class="button" onclick="checkAnswer(true)">Red</button><br>
+            <button class="button" onclick="checkAnswer(false)">Black</button><br>
+            <button class="button" onclick="checkAnswer(false)">Brown</button><br>
+            <button class="button" onclick="checkAnswer(true)">Black and white</button><br>
+            <button class="button" onclick="checkAnswer(false)">White</button><br>
+      <!--  </div> -->
+
+        <script type="text/javascript">
+      
+           function checkAnswer(trueOrFalse){
```

Comment:
Do not name variables after their optional values. what would you do if parameter have lots of options?
Try meaningful names like:  `correct` or `isCorrect`